casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala

```diff
@@ -478,7 +478,7 @@ object BlockAPI {
         tupleSpaceHash = PrettyPrinter.buildStringNoLimit(tsHash),
         tupleSpaceDump = tsDesc,
         timestamp = timestamp,
-        faultTolerance = normalizedFaultTolerance - initialFault,
+        faultTolerance = normalizedFaultTolerance - initialFault, // TODO: Fix
```

Comment:
Fix what? :)

Reply:
It is supposed to be a plus sign but I'll need to type up why.